libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -35,6 +39,12 @@ export class BookSearchComponent implements OnInit {
     this.store.select(getAllBooks).subscribe(books => {
       this.books = books;
     });
+
+    this.searchFormValueChangeSubscription = this.searchForm.get('term').valueChanges.pipe(debounceTime(500)).subscribe(() => this.searchBooks());
```

Comment:
Minor point but this line is really long - are you using Prettier to format your code?

Reply:
Also it would be worth taking a look at the `distinctUntilChanged` operator (or possibly `distinctUntilKeyChanged`)

Reply:
> Minor point but this line is really long - are you using Prettier to format your code?

Thanks for the catch, I was not.